introduction.md

```diff
@@ -1,3 +1,4 @@
-this is Wei's introduction~    
-hello git hub !
-just say Ruby as asked
+* this is Wei's introduction~    
+* hello git hub !
+* just say Ruby as asked
+* Say Git 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?